README.md

```diff
@@ -44,7 +44,7 @@ Pkg.add(PackageSpec(url="https://github.com/holgerteichgraeber/ClustForOpt.jl",
 ```
 Second install `CEP.jl`:
 ```julia
-Pkg.add(PackageSpec(url="https://github.com/YoungFaithful/CEP.jl.git"))
+Pkg.add(PackageSpec(url="https://github.com/YoungFaithful/CEP.jl.git", rev="dev"))
```

Comment:
Checkout "dev" if you want dev

Reply:
How about mentioning this together in both the dev and master README in similar style to the following?

If you like to install `CEP.jl` on the default master branch:
```julia
Pkg.add(PackageSpec(url="https://github.com/YoungFaithful/CEP.jl.git"))
```
If you like to install `CEP.jl` on the development `dev` branch:
```julia
Pkg.add(PackageSpec(url="https://github.com/YoungFaithful/CEP.jl.git", rev="dev"))
```

Reply:
I like that idea :+1: